packages/sources/coinmetrics/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const PAGE_SIZE = 10_000
```

Comment:
Seems like this does not need it's own file

Reply:
If anything, this should be set as a default in the config file, and be modifiable by env vars.